README.md

```diff
@@ -228,12 +228,17 @@ Coverage data can be shown after the execution of a program that was
 compiled to generate this information terminates.  SuperBOL will
 display coverage on a line-by-line basis, by highlighting the lines of
 your source code using colors that represent their coverage status.
+To enable coverage highlighting, you can open the command palette
+(<kbd>Ctrl+Shift+P</kbd>), and select `SuperBOL: Show program coverage`.
```

Comment:
```suggestion
(or type <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>), and select `SuperBOL: Show program coverage`.
```

Reply:
We typeset commands as English titles: with full capitals. So here that should be `SuperBOL: Show Program Coverage` (`SuperBOL: Show Coverage` would be better I think, "program" may not be relevant when working on at module level).

(Same for the other commands below)